include/platform/radio.h

```diff
@@ -121,6 +121,32 @@ typedef enum PhyState
 } PhyState;
 
 /**
+* The radio driver calls this function pointer to notify OpenThread of a received packet.
+*
+* @param[in]  aInstance The OpenThread instance structure.
+* @param[in]  aPacket   A pointer to the received packet or NULL if the receive operation was aborted.
+* @param[in]  aError    ::kThreadError_None when successfully received a frame, ::kThreadError_Abort when reception
+*                       was aborted and a frame was not received.
+*
+*/
+typedef void(*otPlatRadioReceiveDone)(otInstance *aInstance, RadioPacket *aPacket, ThreadError aError);
```

Comment:
please fix these radio api changes... there are no necessary and break support for all silicon vendors that provide radio drivers.